service/debugger/debugger.go

```diff
@@ -1310,6 +1310,23 @@ func (d *Debugger) ListDynamicLibraries() []api.Image {
 	return r
 }
 
+// ExamineMemory returns a list of specific address memory that len is num.
+func (d *Debugger) ExamineMemory(address uintptr, count int) ([]byte, error) {
```

Comment:
s/count/length/ .. I think we use `size` internally in `proc/mem.go` as well, but I think either works a bit btter than `count`.